### PR TITLE
update method for EcoNewsController

### PR DESCRIPTION
**Pull request by Yura09**

### CODE REVIEW

File: core/src/main/java/greencity/dto/econews/UpdateEcoNewsDto.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@Builder
+public class UpdateEcoNewsDto {
+    @NotNull
+    @Min(1)
+    private Long id;
+
+    @Size(min = 1, max = 170)
+    private String title;
+
+    @Size(min = 20, max = 63206)
```
Comment (antonbakalets) [inline core/src/main/java/greencity/dto/econews/UpdateEcoNewsDto.java:31]: This looks too much.

Comment (Yura09) [reply core/src/main/java/greencity/dto/econews/UpdateEcoNewsDto.java:31]: But entity also has the same constraints

**Review by forestik — APPROVED** [review]

**Review by eyouu — APPROVED** [review]

File: core/src/main/java/greencity/dto/econews/UpdateEcoNewsDto.java (hunk 1 above)
Comment (antonbakalets) [reply core/src/main/java/greencity/dto/econews/UpdateEcoNewsDto.java:31]: By the way, is constraint on entity is still working or can be removed? 

You will need to disable this one to check.

Comment (Yura09) [reply core/src/main/java/greencity/dto/econews/UpdateEcoNewsDto.java:31]: done.

**Review by TarasDovganyuk — APPROVED** [review]

**Pull request merged**